### PR TITLE
Fix struct dirent offsets

### PR DESCRIPTION
**Pull request by mrmacete**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -194,31 +207,46 @@ function list(path) {
...
-    while (!((entry = readdir(dirHandle)).isNull())) {
+    while (!((entry = readdirImpl(dirHandle)).isNull())) {
       callback(entry);
     }
   } finally {
     closedir(dirHandle);
   }
 }
 
+function direntReadField(entry, name) {
```
Comment (oleavr) [inline index.js]: Perhaps `readDirentField` would be slightly more consistent? (Verb first.)

**Comment by mrmacete** [comment]
oops, just figured out in linux the offsets are different for 32 bits, fixing now

**Comment by oleavr** [comment]
Sweet!

**Comment by oleavr** [comment]
+ frida-fs@1.3.0

**Pull request merged**